src/locale/locales/pt-BR/messages.po

```diff
@@ -4960,7 +4960,7 @@ msgstr "Post Escondido por Você"
 
 #: src/components/dialogs/PostInteractionSettingsDialog.tsx:283
 msgid "Post interaction settings"
-msgstr ""
+msgstr "Configurações de interação do post"
```

Comment:
In other places you used `postagem` as a translation for `post`

Reply:
ok